app/views/contexts/calculated_summary_context.py

```diff
@@ -16,6 +16,17 @@
 
 
 class CalculatedSummaryContext(Context):
+    def get_return_to(self, section_id: str):
+        # :TODO: Add support for returning to the calculated summary that was used
+        has_section_summary = bool(self._schema.get_summary_for_section(section_id))
+        if has_section_summary:
+            return "section-summary"
```

Comment:
```suggestion
        if has_section_summary := bool(self._schema.get_summary_for_section(section_id)):
            return "section-summary"
```

Reply:
We can add a doctring explaining what we do instead of assigning a var?

Reply:
Do we need one?.

Reply:
If it's for removal I am not precious about it and it can stay.